packages/stem/example/flutter_stem_example/web/index.html

```
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)